trio/socket.py

```diff
@@ -83,7 +83,7 @@
     IP_BIND_ADDRESS_NO_PORT: int
 else:
     try:
-        IP_BIND_ADDRESS_NO_PORT
+        IP_BIND_ADDRESS_NO_PORT  # noqa: B018  # "useless expression"
```

Comment:
Oh yeah, can probably just do a search+replace for `# noqa: B018\n` -> `# noqa: B018 # "useless expression"`

Reply:
Changed in 4213618